src/logging/simpleTest.js

```diff
@@ -0,0 +1,52 @@
+const logModule = require('./logger.js');
```

Comment:
Please move out of src so that code scans don't waste our time on such things.

Reply:
I did now but github had an outage that appears to not notice that I actually did make a new commit: https://github.com/1000TurquoisePogs/zlux-shared/commit/52131883225a10d735c1df077e79529d3e25a946